http/headers/cross-origin-resource-policy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "http": {
+    "headers": {
+      "corb": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy",
+          "support": {
+            "chrome": {
+              "version_added": "73"
+            },
+            "chrome_android": {
+              "version_added": "73"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "12"
+            },
+            "safari_ios": {
+              "version_added": "12"
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "73"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
```

Comment:
Isn't it being standardized?

Reply:
I couldn't find an IETF RFC or W3C draft or standard, as I've seen in the past `SameSite` cookie & CSP. I must admit I'm not sure what WhatWG are really doing nor am I up to date with whether W3C is really relevant anymore.. it's in WhatWG's (unversioned?) "living standard" - does that qualify for `"standard_track": true`?.